samples/boards/nordic/system_off/src/main.c

```diff
@@ -16,7 +16,12 @@
 #include <zephyr/sys/poweroff.h>
 #include <zephyr/sys/util.h>
 
+#if IS_ENABLED(CONFIG_GRTC_WAKEUP_ENABLE)
```

Comment:
https://github.com/zephyrproject-rtos/zephyr/blob/6e653a96f7903fc932575b37f449ceaf4e94c3e7/include/zephyr/sys/util_macro.h#L133-L134

Reply:
Switched to pure `#if defined()`